Marlin/src/module/planner.cpp

```diff
@@ -714,11 +714,11 @@ void Planner::calculate_volumetric_multipliers() {
  *  fr_mm_s     - (target) speed of the move
  *  extruder    - target extruder
  */
-void Planner::_buffer_steps(const int32_t (&target)[XYZE], float fr_mm_s, const uint8_t extruder) {
+void Planner::_buffer_steps(const int32_t (&target)[ABCE], float fr_mm_s, const uint8_t extruder, const float millimeters /*= 0.0*/) {
```

Comment:
Using `const float &millimeters` (const reference) makes argument passing quicker — only 2 bytes instead of 4.